API.md

```diff
@@ -153,19 +159,7 @@ Log.isLog('hello')
 // false
 ```
 
-#### Log.expand(ipfs, log, [amount=-1])
-
-Expands a `log` by `amount` by retreiving more entries from the tails of the log. Returns a new `Log` instance. 
-
-Expanding a log will retrieve new entries from IPFS, thus causing side effects.
-
-#### Log.expandFrom(ipfs, log, entries, [amount=-1])
-
-Expand a `log` by `amount` by retrieving more entries starting from `entries`. `entries` is an `Array` of `Entry` instances. Returns a new `Log` instance. 
-
-Expanding a log will retrieve new entries from IPFS, thus causing side effects.
-
-#### Log.fromEntry(ipfs, entry, [length=-1])
```

Comment:
👍

Reply:
Should also remove these from README "Static Methods" section (and any other references that might still exist).